docker/entrypoint.sh

```diff
@@ -126,42 +126,21 @@ then
   mysql -D $MYSQL_DATABASE -u $MYSQL_USER -p$MYSQL_PASSWORD -h $MYSQL_HOST -P $MYSQL_PORT -e "UPDATE \`setting\` SET \`value\`='$MAINTENANCE_KEY' WHERE \`setting\`='MAINTENANCE_KEY' LIMIT 1"
 fi
 
-if [ -e /CMS-FLAG ]
-then
-  # Remove the CMS-FLAG so we don't run this block each time we're started
-  rm /CMS-FLAG
-
-  # Write settings.php
-  echo "Updating settings.php"
-
-  # We won't have a settings.php in place, so we'll need to copy one in
-  cp /tmp/settings.php-template /var/www/cms/web/settings.php
-  chown apache.apache -R /var/www/cms/web/settings.php
-
-  SECRET_KEY=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 8)
-  /bin/sed -i "s/define('SECRET_KEY','');/define('SECRET_KEY','$SECRET_KEY');/" /var/www/cms/web/settings.php
-fi
-
 if [ "$CMS_DEV_MODE" == "false" ]
 then
-    # Update /etc/periodic/15min/cms-db-backup with current environment (for cron)
-    /bin/sed -i "s/^MYSQL_USER=.*$/MYSQL_USER=$MYSQL_USER/" /etc/periodic/15min/cms-db-backup
-    /bin/sed -i "s/^MYSQL_PASSWORD=.*$/MYSQL_PASSWORD=$MYSQL_PASSWORD/" /etc/periodic/15min/cms-db-backup
-    /bin/sed -i "s/^MYSQL_HOST=.*$/MYSQL_HOST=$MYSQL_HOST/" /etc/periodic/15min/cms-db-backup
-    /bin/sed -i "s/^MYSQL_PORT=.*$/MYSQL_PORT=$MYSQL_PORT/" /etc/periodic/15min/cms-db-backup
-    /bin/sed -i "s/^MYSQL_DATABASE=.*$/MYSQL_DATABASE=$MYSQL_DATABASE/" /etc/periodic/15min/cms-db-backup
+    # Define the secret if not done already
+    SECRET_KEY=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 8) && \
+    /bin/sed -i "s/define('SECRET_KEY','');/define('SECRET_KEY','$SECRET_KEY');/" /var/www/cms/web/settings.php
```

Comment:
It will, but it doesn't matter becasue `define('SECRET_KEY','');` isn't there. It will be replaced on the first run with `define('SECRET_KEY','.........');`